packages/deployment/ansible/roles/prereq/tasks/main.yml

```diff
@@ -20,25 +20,30 @@
     - xfsprogs
     - "nodejs={{ NODEJS_VERSION }}.*"
 
-- name: Stat /dev/sda
+- name: Default home_dev for DigitalOcean
+  set_fact:
+    home_dev: /dev/sda
+  when: "home_dev is not defined and provider is defined and provider.startswith('digitalocean')"
```

Comment:
`is defined`? what language is that? oh... TIL: [Conditionals — Ansible Documentation](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html)